tedge_config/tests/test_tedge_config.rs

```diff
@@ -173,7 +172,7 @@ id = "ABCD1234"
     let config =
         TEdgeConfigRepository::new_with_defaults(config_location, config_defaults).load()?;
 
-    assert_eq!(config.query(DeviceIdSetting)?, "ABCD1234");
```

Comment:
I think we don't need it anymore.
```
[device]
id = "ABCD1234"
"#;
```

Reply:
I can remove it if confusing.

Reply:
I prefer to remove id="ABCD1234" from test cases except `test_any_device_id_provided_by_the_configuration_is_ignored()`.